Extension/src/abTesting.ts

```diff
@@ -8,16 +8,21 @@ import * as util from './common';
 import * as Telemetry from './telemetry';
 import { PersistentState } from './LanguageServer/persistentState';
 import * as fs from 'fs';
+import { PackageVersion } from './packageVersion';
 
 const userBucketMax: number = 100;
 const userBucketString: string = "CPP.UserBucket";
 const localConfigFile: string = "cpptools.json";
+const minimumVSCodeVersionDefault: PackageVersion = new PackageVersion("1.43.2");
```

Comment:
Will need to be updated `minimumVSCodeVersionDefault` whenever extension updates its VSCode version dependency? Is there a way to get this version number from the package.json instead of hardcoding it here?

Reply:
This is a default in case the package.json doesn't contain the version number. This version (i.e 1.43.2) was hardcoded before, so I used it as a default. The version extracted from package.json will replace this.

Reply:
This won't be needed if you use PersistentState for the class field.  You put this value as the default for the PersistentState instead.